### DISCUSSION TITLE
Universal Display Driver - how do you set Option A3?

### DISCUSSION THREAD

**Discussion by Sum-Guy** [arendst/Tasmota#17553] [Q&A] [answered]
Use of the Universal Display Driver requires:

- compiling with #define USE_UNIVERSAL_DISPLAY
-  and setting an unused GPIO to Option A3.

How do you set a GPIO to Option A3?  Where do you set it?

Is that a "setoption" command?  I've seen the setoption n list, where n runs from 1 to 149.  I don't see where n is an alpha-numeric like "A3".  

The "setting an unused GPIO to Option A3" - is that a compiler directive?  Does the bin have to be compiled like that?  Or do I / can I set Option A3 through a console command once tasmota is up and running?

**Comment by sfromis** [comment, marked as answer]
The same way as you configure other gpio components. The dropdown has `Option A` and select number 3 next step.

This way of doing it allows a preconfigured template from the templates repository to include the selection of universal display driver along with the pin config to match, instead of needing both template and additional settings to be done.
